modules/stochastic_tools/test/tests/reporters/statistics/stat_math_test.py

```diff
@@ -0,0 +1,14 @@
+import numpy as np
```

Comment:
Do we need this file on long term?

Reply:
Crap, didn't mean check in any of these python files.